packs/jenkins/actions/disable_job.yaml

```diff
@@ -0,0 +1,12 @@
+---
+name: "disable_job"
+runner_type: "run-python"
+description: "Disable Jenkins Job."
+enabled: true
+entry_point: "disable_job.py"
+parameters:
+  project:
```

Comment:
Actually, looking at the Jenkins docs and Jenkins Python client library docs looks like that this should be the job name and not the project name (https://git.openstack.org/cgit/openstack/python-jenkins/tree/jenkins/__init__.py#n897, http://python-jenkins.readthedocs.io/en/latest/examples.html#example-3-working-with-jenkins-jobs).

I will go ahead and make the change (rename parameter to name, update description).